libc/test/src/stdio/sprintf_test.cpp

```diff
@@ -533,6 +642,10 @@ TEST(LlvmLibcSPrintfTest, OctConv) {
   EXPECT_EQ(written, 3);
   ASSERT_STREQ(buff, "135");
 
+  written = LIBC_NAMESPACE::sprintf(buff, "%3b", 0b111);
```

Comment:
I think you put this in the wrong spot

Reply:
Ah, good catch. Made it early in the morning. 😅 I've pushed an update.